src/main/kotlin/frc/robot/RobotContainer.kt

```diff
@@ -94,6 +110,20 @@ object RobotContainer {
             .onTrue(outtakeAlgae(driverController.L1().negate()))
         driverController.R2().whileTrue(gripper.intake())
         driverController.L2().whileTrue(gripper.outtake())
+        driverController
+            .povLeft()
+            .whileTrue(
+                pathFindToPose(selectedPose)
+                    .andThen(alignToPose(swerveDrive, true, l4()))
+            )
+            .onFalse(l4(Trigger { true }))
```

Comment:
```suggestion
            .onFalse(l4({ true }))
```
I think this will work because `Trigger` inherits `BooleanSupplier`.

Reply:
We checked, it didn't

Reply:
Ok but you can change the type that `l4` takes

Reply:
I don;t think it's necessary

Reply:
It's cleaner and if you can use a "broader" type -- you should. This will make your code more flexible.

Reply:
I don't want to mess with stuff that works. I know it shouldn't mess anything but it's not critical enough to do it 2 days before a competition